Jenkinsfile.github

```diff
@@ -81,14 +81,16 @@ pipeline {
                         SNYK_TOKEN = credentials('SNYK_TOKEN')
                     }
                     steps {
-                        script {
-                            slackSend(channel: env.SLACK_CHANNEL_ALERTS, message: "<${env.BUILD_URL}|CSM ${env.RELEASE_VERSION}> - Running release.sh")
-                            sh """
-                                . build/.env/bin/activate
-                                make clean
-                                rm -fr dist
-                                ./release.sh
-                            """
+                        withCredentials([usernamePassword(credentialsId: credentialsId, usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_TOKEN')]) {
```

Comment:
Just a minor thing - I believe only `sh` call below needs to be encapsulated in `withCredentials`